CMakeLists.txt

```diff
@@ -996,11 +996,15 @@ if(SDL3MIXER_MIDI_NATIVE)
         enable_language(CXX)
         target_sources(${sdl3_mixer_target_name} PRIVATE src/codecs/native_midi/native_midi_haiku.cpp ${midi_common_sources})
         target_link_libraries(${sdl3_mixer_target_name} PRIVATE midi)
+    elseif("${CMAKE_SYSTEM_NAME}" MATCHES "Linux")
+        list(APPEND SDL3MIXER_MIDI_NATIVE_ENABLED TRUE)
+        target_sources(${sdl3_mixer_target_name} PRIVATE src/codecs/native_midi/native_midi_linux_alsa.c ${midi_common_sources})
+        target_link_libraries(${sdl3_mixer_target_name} PRIVATE asound)
```

Comment:
I think linking to `asound` might be an issue: not everybody has alsa installed. It should also be possible to load alsa dynamically (using `SDL_LoadObject`).
Also, should we call alsa "native"? Isn't it "just" a library, that happens to be Linux-only, just like pipewire? Perhaps define a new `SDL3MIXER_MIDI_ALSA` option?
```cmake
cmake_dependent_option(SDL3MIXER_MIDI_ALSA "Support for playing MIDI with alsa-lib" ON "SDL3MIXER_MIDI;UNIX;NOT APPLE" OFF)`
cmake_dependent_option(SDL3MIXER_MIDI_ALSA_SHARED "Load alsa-lib dynamically" ON "SDL3MIXER_MIDI_ALSA" OFF)`
```
With those changes, I'll help with the cmake changes.

Reply:
> Isn't it "just" a library, that happens to be Linux-only, just like pipewire?

ALSA (and in this case `snd_seq.*`) is a kernel module/implementation. libasound is a library to make talking to the kernel easier. PipeWire is a userspace daemon on the other hand that also depends on ALSA support in the kernel (to output audio to physical ports), and runs on top of it as that is how the Linux kernel itself implements audio.

Reply:
Indeed, it's a "helper library", so not guaranteed to be installed on a system (forget about my remark about pipewire).